### PR TITLE
Add WCOSS2 support in configs and setup scripts

### PR DESCRIPTION
**Pull request by KateFriedman-NOAA**
- Add needed WCOSS2 support in configs and setup scripts. Tested setup scripts successfully on Cactus (have not yet tested xml).
- Created and added WCOSS2.env file. Have not yet tested at runtime.

Refs: #399

### CODE REVIEW

File: env/WCOSS2.env [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+export npe_node_max=128
+
+# Configure MPI environment
+export MPI_LABELIO=YES
+export MP_STDOUTMODE="ORDERED"
+export KMP_STACKSIZE=2048M
+export KMP_AFFINITY=scatter
+export job=${LSB_JOBNAME:-$step}
+export jobid=${job}.${LSB_JOBID:-$$}
```
Comment (aerorahul) [inline env/WCOSS2.env]: Are you should this is `LSB_JOBID`?  This is a PBS scheduler.

File: env/WCOSS2.env [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+export mpmd="--cpu-bind verbose,core cfp"
+
+export npe_node_max=128
+
+# Configure MPI environment
+export MPI_LABELIO=YES
+export MP_STDOUTMODE="ORDERED"
+export KMP_STACKSIZE=2048M
+export KMP_AFFINITY=scatter
+export job=${LSB_JOBNAME:-$step}
```
Comment (aerorahul) [inline env/WCOSS2.env]: Are you should this is `LSB_JOBNAME`? This is a PBS scheduler.

**Review by aerorahul — COMMENTED** [review]
looks ok to me.
just a couple of comments.

File: env/WCOSS2.env (hunk 1 above)
Comment (KateFriedman-NOAA) [reply env/WCOSS2.env]: Good catch, this should be a PBS variable, have now fixed, thanks!

File: env/WCOSS2.env (hunk 2 above)
Comment (KateFriedman-NOAA) [reply env/WCOSS2.env]: Fixed, thanks!

**Comment by KateFriedman-NOAA** [comment]
@aerorahul I believe I have addressed your comments, thanks!

File: env/WCOSS2.env [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+export launcher="mpiexec -l"
+export mpmd="--cpu-bind verbose,core cfp"
+
+export npe_node_max=128
+
+# Configure MPI environment
+export MPI_LABELIO=YES
+export MP_STDOUTMODE="ORDERED"
+export KMP_STACKSIZE=2048M
+export KMP_AFFINITY=scatter
```
Comment (aerorahul) [inline env/WCOSS2.env:26]: This is fine for now, but we should ask the system owner if these are advantageous or not needed on this machine.

**Review by aerorahul — APPROVED** [review]
looks good to me.

File: env/WCOSS2.env (hunk 3 above)
Comment (KateFriedman-NOAA) [reply env/WCOSS2.env:26]: Definitely...these are what is used on the Dells right now so I need inputs from others on what (if any) to set here.

**Comment by KateFriedman-NOAA** [comment]
Thanks @aerorahul !

**Pull request merged**